### PR TITLE
Added mobile resolution scripts | Removed hardcoded url to use DEPLOY_HOST | Added pom selector for mobile

### PR DESCRIPTION
**Pull request by avozicov**

### CODE REVIEW

File: test/e2e/tests/mobile/assert-mobile-menu.e2e.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        portraitOrientation: true
+      });
+  });
+
+test('Check "Top Menu" with included links is visible for mobile view and can be expanded by the click', async t => {
+  await t
+    .expect(home.mobileMenu.exists).ok()
+    .hover(home.mobileMenu)
+    .click(home.mobileMenu, { speed: 0.5 });
+    await t
```
Comment (grigoreme) [inline test/e2e/tests/mobile/assert-mobile-menu.e2e.js]: Please use empty line between functions

File: test/e2e/tests/mobile/assert-mobile-menu.e2e.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    .expect(header.challengesTopMenuLink.exists).ok()
+    .expect(header.challengesTopMenuLink.innerText).contains('CHALLENGES');
+    await t
```
Comment (grigoreme) [inline test/e2e/tests/mobile/assert-mobile-menu.e2e.js]: same

File: test/e2e/tests/mobile/assert-mobile-menu.e2e.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    .expect(header.solutionsTopMenuLink.exists).ok()
+    .expect(header.solutionsTopMenuLink.innerText).contains('SOLUTIONS');
+    await t
```
Comment (grigoreme) [inline test/e2e/tests/mobile/assert-mobile-menu.e2e.js]: same

File: test/e2e/tests/mobile/assert-mobile-menu.e2e.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    .expect(header.apiTopMenuLink.exists).ok()
+    .expect(header.apiTopMenuLink.innerText).contains('API');
+    await t
+    .expect(header.teamTopMenuLink.exists).ok()
+    .expect(header.teamTopMenuLink.innerText).contains('TEAM');
+    await t
+    .expect(header.contactTopMenuLink.exists).ok()
+    .expect(header.contactTopMenuLink.innerText).contains('CONTACT');
+    await t
```
Comment (grigoreme) [inline test/e2e/tests/mobile/assert-mobile-menu.e2e.js]: same

File: test/e2e/tests/mobile/mobile-request-demo.e2e.js [hunk 5]
```diff
@@ -34,4 +32,4 @@ test('Check user can submit "Request a demo" form with valid email address', asy
       visibilityCheck: true,
     }).visible).ok()
     .expect(Selector(requestDemoModal.responseText).innerText).contains('\nThank you for the interest in AdTechMedia WordPress Plugin\nPlease check your inbox to schedule your demo.\n');
```
Comment (grigoreme) [inline test/e2e/tests/mobile/mobile-request-demo.e2e.js]: Get rid of that big line, you can split into new lines by functions like:
```
expect(
  Selector(requestDemoModal.responseText).innerText)
  .contains('\nThank you for the interest in AdTechMedia WordPress Plugin\nPlease check your inbox to schedule your demo.\n');```

**Review by grigoreme — CHANGES_REQUESTED** [review]

File: test/e2e/tests/mobile/assert-mobile-menu.e2e.js (hunk 1 above)
Comment (avozicov) [reply test/e2e/tests/mobile/assert-mobile-menu.e2e.js]: got it, thank you.
Fixed.

File: test/e2e/tests/mobile/mobile-request-demo.e2e.js (hunk 5 above)
Comment (avozicov) [reply test/e2e/tests/mobile/mobile-request-demo.e2e.js]: got it, thank you.
Fixed.

**Review by grigoreme — APPROVED** [review]

**Pull request merged**